tests/suite/resources_utils.py

```diff
@@ -914,3 +914,19 @@ def get_events(v1: CoreV1Api, namespace) -> []:
     print(f"Get the events in the namespace: {namespace}")
     res = v1.list_namespaced_event(namespace)
     return res.items
+
+
+def ensure_response_from_backend(req_url, host) -> None:
+    """
+    Wait for 502 to disappear.
+
+    :param req_url: url to request
+    :param host:
+    :return:
+    """
+    for _ in range(0, 5):
```

Comment:
```suggestion
    for _ in range(5):
```
The `from` in range is 0 by default I think

Reply:
Right, I'll update